content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -282,6 +282,11 @@ on the Kubernetes API server for each static Pod.
 This means that the Pods running on a node are visible on the API server,
 but cannot be controlled from there.
 
+{{< note >}}
+Static Pods do not support {{< glossary_tooltip text="ConfigMap" term_id="configmap" >}} 
+or {{< glossary_tooltip text="Secret" term_id="secret" >}}.
```

Comment:
This is too glib about ConfigMap and Secret and omits the larger issue with ServiceAccounts.  Perhaps something like the following would be better.

> The definition of a Static Pod may not refer to a {{< glossary_tooltip text="ConfigMap" term_id="configmap" >}} or a {{< glossary_tooltip text="Secret" term_id="secret" >}} and may not have a {{< glossary_tooltip text="ServiceAccount" term_id="service-account" >}}.

Reply:
I have discovered that the prohibition is broader and simpler.

> The definition of a Static Pod may not refer to any other API object.